stringfuzz/transformers/translate.py

```diff
@@ -0,0 +1,46 @@
+'''
+Permuting the alphabet in every string literal.
+'''
+
+import random
+import string
+import copy
+
+from stringfuzz.ast import StringLitNode
+from stringfuzz.ast_walker import ASTWalker
+from stringfuzz import ALL_CHARS
+
+__all__ = [
+    'translate'
+]
+
+WITH_INTEGERS    = list(ALL_CHARS)
+WITHOUT_INTEGERS = [c for c in ALL_CHARS if not c.isdecimal()]
+
+class TranslateTransformer(ASTWalker):
+    def __init__(self, ast, character_set):
+        super(TranslateTransformer, self).__init__(ast)
+        shuffled = copy.copy(character_set)
+        random.shuffle(shuffled)
+
+        shuffled      = ''.join(shuffled)
+        character_set = ''.join(character_set)
```

Comment:
This reads a little weirdly, partly because `character_set` is changing types in this assignment. What do you think of encapsulating these `join`s and the `maketrans` into a `make_table` function that takes two character lists and returns a table?

Reply:
Sounds good!